cmd/goal/node.go

```diff
@@ -116,6 +117,7 @@ func init() {
 
 	catchupCmd.Flags().BoolVarP(&abortCatchup, "abort", "x", false, "Aborts the current catchup process")
 	catchupCmd.Flags().BoolVar(&fastCatchupForce, "force", false, "Forces fast catchup with implicit catchpoint to start without a consent prompt")
+	catchupCmd.Flags().Uint64VarP(&initializeCatchup, "initialize", "i", 0, "Catchup only if the catchpoint would advance the node by at least the specified number of rounds")
```

Comment:
maybe call it something like "minAdvanceRounds" or similar? IMO `initialize` does not tell anything.

Reply:
I'll rename in a followup PR.